### ISSUE TITLE
type annotations

### ISSUE THREAD

**Issue by seanmacavaney** [terrierteam/ir_measures#5]
add type annotations throughout

**Comment by heinrichreimer** [comment]
Yep, that would indeed be great! In #43 I have added two very simple types for `Qrels` and `Run`:

https://github.com/terrierteam/ir_measures/blob/27088215789dad877f75b9aba4d456e1894556ec/ir_measures/util.py#L31
https://github.com/terrierteam/ir_measures/blob/27088215789dad877f75b9aba4d456e1894556ec/ir_measures/util.py#L40

We could probably also include `Iterable[NamedTuple]` in the union so that it matches the documented behavior of `QrelsConverter` and `RunConverter`.
